README.md

```diff
@@ -2,3 +2,5 @@
 Sample program to start with akka actors framework. 
 
```

Comment:
The program description is not clear. It would be helpful to provide more details about what the program does and how it works.